configure.py

```diff
@@ -1051,7 +1066,10 @@ def is_arm_hard_float_abi():
 def host_arch_cc():
   """Host architecture check using the CC command."""
 
-  k = cc_macros(os.environ.get('CC_host'))
+  if sys.platform.startswith('zos'):
+    return 's390x';
+  else:
+    k = cc_macros(os.environ.get('CC_host'))
```

Comment:
The `else` seems to be redundant.

```suggestion
  k = cc_macros(os.environ.get('CC_host'))
```